forem/forem
# Clear asset and vendor caches on deploy

## What type of PR is this? (check all applicable)


## Description

We don't need these files inside deployment artifacts. They're only there to make asset compilation and bundling faster. This should give us a lot more headroom between our Heroku slug sizes and the soft cap of 300MB.

## Related Tickets & Documents

#13563
[Forem Slack thread](https://forem-team.slack.com/archives/CPGJMAX5F/p1620050711348800?thread_ts=1620049708.343800&cid=CPGJMAX5F)

## QA Instructions, Screenshots, Recordings

Check [build logs](https://dashboard.heroku.com/apps/benhalpern-community/activity) for [BHC](https://community.benhalpern.com). They should show slug size much smaller — last one I deployed manually was 292MB.

The latest build for DEV as of this writing is 322MB, which is above the soft cap.

## Added tests?

- [x] No, and this is why: No runtime code is affected

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

![Max Headroom](https://target.scene7.com/is/image/Target/GUEST_2a39a9ed-2aa3-42f8-be38-33bfab673020?wid=488&hei=488&fmt=pjpeg)
